rpc/v1/openapi.yaml

```diff
@@ -130,110 +218,1622 @@ paths:
+        - signature
+        - address
+      properties:
+        signature:
+          type: string
+        address:
+          type: string
+    Payload:
+      type: object
+      required:
+        - data
+        - method
+        - path
+        - headers
+      properties:
+        data:
+          type: string
+        method:
+          type: string
+        path:
+          type: string
+        headers:
+          $ref: "#/components/schemas/Headers"
+    Pool:
+      type: object
+      required:
+        - address
+        - name
+        - amount
+        - denom
+      properties:
+        address:
+          type: string
+        name:
+          type: string
+        amount:
+          type: string
+        denom:
+          type: string
+    QuorumCertificate:
+      type: object
+      required:
+        - height
+        - round
+        - step
+        - block
+        - threshold_sig
+        - transactions
+      properties:
+        height:
+          type: integer
+          format: int64
+        round:
+          type: integer
+          format: int64
+        step:
+          type: string
+        block:
+          type: string
+        threshold_sig:
+          $ref: "#/components/schemas/ThresholdSignature"
+        transactions:
+          type: array
+          items:
+            type: string
+    RelayRequestMeta:
+      type: object
+      required:
+        - block_height
+        - servicer_pub_key
+        - chain
+        - geozone
+        - token
+        - signature
+      properties:
+        block_height:
+          type: integer
+          format: int64
+        servicer_pub_key:
+          type: string
+        chain:
+          $ref: "#/components/schemas/Identifiable"
+        geozone:
+          $ref: "#/components/schemas/Identifiable"
+        token:
+          $ref: "#/components/schemas/AAT"
+        signature:
+          type: string
+    Signature:
+      type: object
+      required:
+        - public_key
+        - signature
+      properties:
+        public_key:
+          type: string
+        signature:
+          type: string
+    StdTx:
+      type: object
+      required:
+        - fee
+        - message
+        - nonce
+        - signature
+      properties:
+        fee:
+          $ref: "#/components/schemas/Fee"
+        message:
+          oneOf:
+            - $ref: "#/components/schemas/MessageSend"
+            - $ref: "#/components/schemas/MessageStake"
+            - $ref: "#/components/schemas/MessageEditStake"
+            - $ref: "#/components/schemas/MessageUnstake"
+            - $ref: "#/components/schemas/MessageUnpause"
+            - $ref: "#/components/schemas/MessageChangeParameter"
+        nonce:
+          type: string
+        signature:
+          $ref: "#/components/schemas/Signature"
+    Transaction:
+      type: object
+      required:
+        - hash
+        - height
+        - index
+        - txResult
+        - stdTx
+      properties:
```

Comment:
I'm beginning to get a bit confused here so I think finding as much consistency between what we have in the code and here will really help.

Note how in the code a `txResult` contains a `Transaction` but here its the opposite. I also want to eliminate `StdTx` because I'm not sure what that means or refers to.

```proto
message Transaction {

  // The message to be signed is intentionally an `Any` type, since it is up to the module to
  // define the exact message type, its contents and validation protocol.
  google.protobuf.Any msg = 1;

  // A one-time nonce to avoid replay previous transaction.
  string nonce = 2;

  // The signature must sign the `Transaction` protobuf containing both the `msg` and `nonce` with
  // a nil signature.
  Signature signature = 3; // The signature
}
```

```proto
message TxResult {
  bytes tx = 1; // a serialized `Transaction` proto
  int64 height = 2; // the block height at which the transaction was included
  int32 index = 3; // the transaction's index within the block (i.e. ordered by when the proposer received it in the mempool)
  int32 result_code = 4; // 0 is no error, otherwise corresponds to error object code; // IMPROVE: Consider using enums for the result codes
  string error = 5; // description of the error if the result code is non-zero; IMPROVE: Add a specific type for the error code
  string signer_addr = 6; // the address of the signer (e.g. sender) of the transaction
  string recipient_addr = 7; // Optional: the address of the recipient of the transaction (if applicable)
  string message_type = 8; // the message type contained in the transaction; must correspond to a proto that the node can can process (e.g. Stake, Unstake, Send, etc...) // IMPROVE: How do we document all the types?
}
```